grafana-plugin/src/components/Integrations/IntegrationTag.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { Tag } from 'components/Tag/Tag';
+import { Text } from 'components/Text/Text';
+
+interface IntegrationTagProps {
+  children: React.ReactNode;
+}
+
+export const IntegrationTag: FC<IntegrationTagProps> = ({ children }) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Tag className={styles.tag}>
+      <Text type="primary" size="small" className={styles.radius}>
+        {children}
+      </Text>
+    </Tag>
+  );
+};
+
+export const getStyles = (theme: GrafanaTheme2) => ({
+  tag: css({
```

Comment:
Could you please move these to a separate file so that we'll have the same pattern across the project? I know it's just 2 classes but it enforces consistency this way for future migration

Reply:
I would like to talk about it and reconsider the "styles always in separate file" rule. The thing is if you have small component, it can be cumbersome to every time be forced to create another file for example just for 1-2 classes, add import etc. https://github.com/grafana/grafana/blob/main/contribute/style-guides/styling.md Here you can see that styles are in the same file. 

I'd like to propose that we extract styles to separate file if:
- the component file itself is big enough (which we should avoid)
- you share styles between different components placed in different files

What do you think?